locations/jclouds/src/main/java/org/apache/brooklyn/location/jclouds/JcloudsLocation.java

```diff
@@ -2297,7 +2297,6 @@ protected JcloudsWinRmMachineLocation createWinRmMachineLocation(ComputeService
                     .configure("jcloudsParent", this)
                     .configure("displayName", vmHostname)
                     .configure("address", address)
-                    .configure(WinRmMachineLocation.WINRM_PORT, sshHostAndPort.isPresent() ? sshHostAndPort.get().getPort() : node.getLoginPort())
```

Comment:
Keep this, used by `WinRmMachineLocation.getPort()`. The override (port forwarding) port would be lost otherwise.